app/presenters/subscription_presenter.rb

```diff
@@ -21,6 +21,19 @@ def to_row
     full_search_criteria.transform_values! { |value| value.is_a?(Array) ? value.join(', ') : value }
   end
 
+  def edit_url(source: nil, medium: nil, campaign: nil, content: nil)
```

Comment:
generation of links is more something for a view helper than a presenter in my opinion